<?xml version="1.0"?>

<launch>
  <arg name="AD0"       default="false"/>

  <!-- rusrun mpu6050 mpu6050_node -->
  <node pkg="mpu6050" type="mpu6050_node" name="mpu6050_node" respawn="true" output="screen">
  <param name="AD0" type="bool" value="$(arg AD0)" />
  <param name="PublishTF" type="bool" value="true" />
  <param name="UseCalibration" type="bool" value="true" />
  <param name="CalibrationTimes" type="int" value="2000" />
  <param name="UseFilter" type="bool" value="true" />
  <param name="FilterGain" type="double" value="0.031" />
  <param name="AccelerationScale" type="int" value="0" />
  <param name="GyroScale" type="int" value="0" />
  <param name="Hz" type="int" value="60" />
  <param name="ParentsframeID" type="string" value="base_link" />
  <param name="FrameID" type="string" value="imu_link" />
</node>

</launch>
